src/Pages/ProductList/ProductList.scss

```
@@ -0,0 +1,60 @@
+@import "./../../Styles/common.scss";
+
+* {
+	box-sizing: border-box;
+}
+
+.ProductList {
+	font-family: "Helvetica Now", "Helvetica Neue", Helvetica, Arial, sans-serif;
+	@include columnCenter;
+	width: 100%;
+	.productContainer {
+		width: 1200px;
+		padding: 0 30px;
+	}
+
+	header {
+		margin-top: 20px;
+		ul {
+			display: flex;
+			padding-bottom: 40px;
+			font-size: 11px;
+			li {
+				&:after {
+					margin: 0 5px 0 7px;
```

```suggestion
					margin-right: 5px;
					margin-left: 7px;
```
이렇게 써주는게 이 경우에는 더 직관적으로 읽히네요!